plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java

```diff
@@ -50,11 +51,7 @@ public MavenAppEngineStandardWizardPage() {
     super("basicNewProjectPage"); //$NON-NLS-1$
     setTitle("Maven-based App Engine Standard Project");
     setDescription("Create new Maven-based App Engine Standard Project");
-    
-    // todo: pull this from commons
-    ImageDescriptor descriptor = AbstractUIPlugin.imageDescriptorFromPlugin(
-        "com.google.cloud.tools.eclipse.appengine.localserver", "icons/gcp-32x32.png"); //$NON-NLS-1$ //$NON-NLS-2$
-    this.setImageDescriptor(descriptor);
+    setImageDescriptor(AppEngineImages.googleCloudPlatform(32));
```

Comment:
Again, I do not think that the AppEngineImages class should decide what size is needed based on a context. That putz the  knowledge in the wrong location and tightly couples this package to its clients. For loosest coupling, all AppEngineImages (and the ui package) should know is which images are available. That's it. The caller is the one that knows what size it wants. 

If we can't provide a resolution independent vector graphic, then the caller has to explicitly or implicitly request an image at a particular size.